providers/lever/supports.go

```diff
@@ -8,17 +8,8 @@ import (
 	"github.com/amp-labs/connectors/internal/components"
 )
 
```

Comment:
Please ensure that all future PRs with this file name it `support.go`. If any other connectors currently call it `supports.go`, please update that in a future PR as well, thanks